db/schema.rb

```diff
@@ -600,7 +600,6 @@
   create_table "images", id: :integer, charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
     t.integer "image_data_id"
     t.integer "edition_id"
-    t.string "alt_text"
```

Comment:
This change will need to follow as a separate PR, otherwise we'll cause an incident - see https://docs.google.com/document/d/1GsLZTyuFBaAyt-VNFk2guV2XIzCoui1KNfe2_gN0lX4/edit?tab=t.0#heading=h.p99426yo0rbv.